main.go

```diff
@@ -102,308 +59,86 @@ func exportEnvironmentWithEnvman(keyStr, valueStr string) error {
 	return cmd.Run()
 }
 
-// -----------------------
-// --- Main
-// -----------------------
 func main() {
-	//
-	// Validate options
-	configs := createConfigsModelFromEnvs()
-
-	fmt.Println()
-	configs.print()
-
-	if err := configs.validate(); err != nil {
-		log.Errorf("Issue with input: %s", err)
-		os.Exit(1)
+	if err := initConfig(); err != nil {
```

Comment:
Instead of using a `fail` func or similar in the `main`, I'd suggest to rewrite main to this pattern:

```
func main() {
	if err := doMain(); err != nil {
		log.Errorf("ERROR: %+v", err)
		os.Exit(1)
	}
	log.Donef("\nSuccess")
}
```

Where the actual implementation is in "`doMain`", which returns error (should NOT os.Exit), and the `main()` is only responsible for proper, unified logging of the error.